src/graphql/types/abstract/transaction.ts

```diff
@@ -34,10 +39,14 @@ export const transactionInterfaceFields = {
       return {
         formattedAmount: settlementUsdPerSatInCents.toString(),
         base: Math.round(settlementUsdPerSatInCents * 10 ** 4),
+        //TODO: What is offset?  What is difference between offset + sats/usd?
         offset: 4,
+        //TODO: Is current offset 0.000001 to the dollar which would be 4 for offset and 2 for usd/cent?
+        // Why not use offset 6 and USD?
```

Comment:
the reason we are using cents and not dollar is to work around the limitation of graphql int. ie: to not overflow.